pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/PulsarClientTool.java

```diff
@@ -110,14 +111,15 @@ public PulsarClientTool(Properties properties) {
 
         produceCommand = new CmdProduce();
         consumeCommand = new CmdConsume();
+        generateDocument = new CmdGenerateDocument();
 
         this.commandParser = new JCommander();
         this.usageFormatter = new DefaultUsageFormatter(this.commandParser);
-
         commandParser.setProgramName("pulsar-client");
         commandParser.addObject(this);
         commandParser.addCommand("produce", produceCommand);
         commandParser.addCommand("consume", consumeCommand);
+        commandParser.addCommand("document", generateDocument);
```

Comment:
What about 'generate_documentation ' ?

Reply:
Good idea, I will revise it.
Actually, I don’t know how to integrate it into the web page. But now there must be automated tools: http://pulsar.apache.org/tools/pulsar-admin/2.7.0-SNAPSHOT/
cc @codelipenghui